openpype/hosts/maya/plugins/publish/validate_skeleton_top_group_hierarchy.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""Plugin for validating naming conventions."""
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import (
+    ValidateContentsOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+
+
+class ValidateSkeletonTopGroupHierarchy(pyblish.api.InstancePlugin,
+                                        OptionalPyblishPluginMixin):
+    """Validates top group hierarchy in the SETs
+    Make sure the object inside the SETs are always top
+    group of the hierarchy
+
+    """
+    order = ValidateContentsOrder + 0.05
+    label = "Top Group Hierarchy"
+    families = ["rig"]
+
+    def process(self, instance):
+        invalid = []
+        skeleton_data = instance.data.get(("animated_rigs"), [])
+        skeletonMesh_data = instance.data(("skeleton_mesh"), [])
```

Comment:
Typos? Are we sure this validator is working and is not listed under "crashed plugins" in the publisher?
```suggestion
        skeleton_data = instance.data.get("animated_rigs"), [])
        skeletonMesh_data = instance.data("skeleton_mesh"), [])
```

Reply:
Also `skeletonMesh_data` variable name should be `skeleton_mesh_data`?